homeassistant/components/solarlog/const.py

```diff
@@ -102,7 +101,7 @@ class SolarLogSensorEntityDescription(SensorEntityDescription):
         name="yield total",
         icon="mdi:solar-power",
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
-        state_class=STATE_CLASS_TOTAL_INCREASING,
+        state_class=STATE_CLASS_TOTAL,
```

Comment:
Assuming the total yield sensor never resets, the state class should be set to `total` without `last_reset`